src/content/docs/teams.mdx

```diff
@@ -8,39 +8,38 @@ collaborators to an app. Shorebird free "Hobby" users can upgrade
 to the Teams tier to add collaborators via a link in the
 [Shorebird console](https://console.shorebird.dev).
 
-## Permissions
+## Roles
 
-Permissions are internally stored per-app. Not all permissions are publicly
-editable at this time, we plan to expose more controls soon. If you would
-like to make changes to your app in the meantime, please do not
-hesitate to reach out to contact@shorebird.dev.
+_Roles are per-app._ Not all roles are publicly editable yet. If you
+have any questions about access control for your app, please do not hesitate to
+reach out to contact@shorebird.dev.
 
-### Collaborator (aka "Developer")
+### App Developer (aka Collaborator)
 
-The Collaborator role can:
+The Developer role can:
 
-- Read app information
-- View collaborators
-- Create releases and patches
-- View insights
+- Read information for a single app
+- View collaborators on that app
+- Create releases and patches for that app
+- View insights for that app
 
-Collaborators can be added and removed by any Admin, see below.
+Developer can be added and removed by any Admin, see below.
 
-### Admin (automatically assigned to the account which created the app)
+### App Admin
 
 The Admin role can:
 
 - Everything a Developer can do
-- Create and Delete Apps
-- Add and Remove Collaborator for an App
+- Add and Remove Developers for an App
+- Delete an App
 
-Admin permissions cannot currently be modified directly. Please email us
+Admin permissions cannot currently be modified in the console. Please email us
 if you need to add/remove Admins from an app.
 
-### Billing account (aka App Owner)
```

Comment:
This is a bit confusing imo because the billing account is completely orthogonal to the collaborators/roles 🤷 
Might make more sense to have billing be it's own section?

Reply:
I've split it out.  PTAL.